src/core/APIWrapper.ts

```diff
@@ -66,15 +68,17 @@ export default class APIWrapper<T_ResponseValidator extends t.Any, T_PostBodyVal
 	constructor(config: Config<T_ResponseValidator, T_PostBodyValidator, T_FixedParams>) {
 		this.config = config;
 	}
+	send: (data: PostType<t.TypeOf<T_PostBodyValidator>>) => Promise<ApiResult<t.TypeOf<T_ResponseValidator>>> = data => this.sendWithParams(none)(data)
```

Comment:
I added a send() method in release/next that takes no arguments, for get calls and posts with no body.  Everything else should use sendJson or sendFormUrlEncoded if it has a body